nginx-letsencrypt/nginx.conf

```diff
@@ -1,34 +1,57 @@
-events { worker_connections 1024; }
+events {
+	worker_connections 1024;
+}
+
 http {
-        server {
+	# language server websockets
+	map $http_upgrade $connection_upgrade {
+		default upgrade;
+		'' close;
+	}
 
-	    listen              80;
-	    server_name         ___server_names___;
+	server {
+		listen 80;
+		server_name subliiime.com;
 
-	    location ^~ /.well-known/acme-challenge {
+		location ^~ /.well-known/acme-challenge {
 			proxy_pass http://sublime_nginx_letsencrypt:80;
-			proxy_set_header Host            $host;
+			proxy_set_header Host $host;
 			proxy_set_header X-Forwarded-For $remote_addr;
 			proxy_set_header X-Forwarded-Proto https;
-	    }
-	    location /v1 {
+		}
+
+		location /v1 {
 			proxy_pass http://sublime_mantis:8000;
-			proxy_set_header   X-Real-IP $remote_addr;
-			proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
-			proxy_set_header   X-Forwarded-Host $server_name;
-	    }
-	    location /v0 {
+			proxy_set_header Host $host;
+			proxy_set_header X-Real-IP $remote_addr;
+			proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+			proxy_set_header X-Forwarded-Host $server_name;
+
+			# language server websockets
+			proxy_set_header Upgrade $http_upgrade;
+			proxy_set_header Connection $connection_upgrade;
+		}
+
+		location /v0 {
 			proxy_pass http://sublime_mantis:8000;
-			proxy_set_header   X-Real-IP $remote_addr;
-			proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
-			proxy_set_header   X-Forwarded-Host $server_name;
-	    }
-	    location / {
+			proxy_set_header Host $host;
+			proxy_set_header X-Real-IP $remote_addr;
+			proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+			proxy_set_header X-Forwarded-Host $server_name;
+		}
+
+		location / {
 			proxy_pass http://sublime_dashboard;
-			proxy_set_header   Host $host;
-			proxy_set_header   X-Real-IP $remote_addr;
-			proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
-			proxy_set_header   X-Forwarded-Host $server_name;
-	    }
+			proxy_set_header Host $host;
+			proxy_set_header X-Real-IP $remote_addr;
+			proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+			proxy_set_header X-Forwarded-Host $server_name;
+		}
+
+		listen 443 ssl; # managed by Certbot
+		ssl_certificate /etc/letsencrypt/live/subliiime.com/fullchain.pem; # managed by Certbot
+		ssl_certificate_key /etc/letsencrypt/live/subliiime.com/privkey.pem; # managed by Certbot
+		include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
+		ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot
```

Comment:
looks like this was accidentally committed

Reply:
@jkamdjou whoops, fixed!